src/main.rs

```diff
@@ -51,10 +47,15 @@ fn optimize() {
     env_logger::init();
 
     // Reading input graph and rules
+    /*
```

Comment:
Remove? If you want to keep this code because it is handy, I would recommend having a proper parameter/flag to control how the input graph is being constructed.

Reply:
Added command line arg parser